types/react-paginate/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-paginate 6.2
+// Type definitions for react-paginate 7.1
```

Comment:
I see a mistake here, because, as far as I understand, for types correct work with version 6 it is necessary to create folder with name "v6", and put there the corresponded types.

And only after that you would be able to change types in root folder - types/react-paginate/index.d.ts.

Moreover there are new major changes in this package which should be added to types definitions. I mean new prop - "onPageActive()" , AdeleD/react-paginate#349

Reply:
I made the v6 folder and I looked over the props for v7 and added all of the missing ones and added a deprecation notice for one.